packages/app-staking/src/Overview/index.tsx

```diff
@@ -20,50 +20,48 @@ interface Props extends ApiProps, BareProps, ComponentProps {
   chain_subscribeNewHeads?: HeaderExtended;
 }
 
-class Overview extends React.PureComponent<Props> {
-  public render (): React.ReactNode {
-    const { chain_subscribeNewHeads, allControllers, allStashes, currentValidatorsControllersV1OrStashesV2, isSubstrateV2, recentlyOnline } = this.props;
-    let nextSorted: string[];
+function Overview (props: Props): React.ReactElement<Props> {
+  const { chain_subscribeNewHeads, allControllers, allStashes, currentValidatorsControllersV1OrStashesV2, isSubstrateV2, recentlyOnline } = props;
```

Comment:
Make a TODO here - we probably want to move this into `useState` - it doesn't need to get re-calculated each run